### PR TITLE
fix issue 100 & 93 &more

### PR DESCRIPTION
**Pull request by CS6**
fix issue 100 & 93 &more

＃issue 100
Center the Main Header title
but [`Fix that the lower part of the countdown animation overlap with the dividing line`] in  PR #110 

# issue 93
The exclamation mark should be replace with a warning icon.

The icon should only show if the highest body temperature of the day exceeds 37.5 Celsius.

＝＝＝＝＝＝＝more＝＝＝＝＝＝＝＝＝
Fixed preview picture size
Fixed "body temperature" text size

### CODE REVIEW

**Comment by CS6** [comment]
image

image
image

File: src/app/daily/daily-overview/daily-overview.component.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ion-thumbnail {
+        width:100%;
+        height:100%;
+
```
Comment (bafu) [inline src/app/daily/daily-overview/daily-overview.component.scss:4]: Remove the unexpected newlines and spaces.

File: src/app/daily/daily-overview/daily-overview.component.html [hunk 2]
```diff
@@ -24,23 +24,32 @@
...
           </ion-col> -->
-             <ion-col size="8">
-            <div>
+             <ion-col size="7">
+              <ion-thumbnail >
               <ion-img *ngIf="!item.imgSrc" src="/assets/imgA.png" alt="No picture"></ion-img>
               <ion-img *ngIf="item.imgSrc" [src]="item.imgSrc" alt="No picture"></ion-img>
-            </div>
+              </ion-thumbnail>
+
```
Comment (bafu) [inline src/app/daily/daily-overview/daily-overview.component.html:52]: Remove the unexpected newline.

**Comment by CS6** [comment]
fixed

**Comment by bafu** [comment]
I've confirmed that issue #93 is fixed by this PR.

File: src/app/daily/daily-overview/daily-overview.component.html [hunk 3]
```diff
@@ -5,42 +5,37 @@
...
+          <ion-col size="5">
             <ion-card-header>
               <ion-card-title class="ion-text-center">DAY {{ item.day }}</ion-card-title>
               <ion-card-title class="ion-text-center">{{ item.month }}/{{ item.date }}</ion-card-title>
             </ion-card-header>
-            <ion-card-content style="font-size: small;" class="ion-text-center">BT: {{ item.bt }}</ion-card-content>
+            <ion-card-content style="font-size: small;" class="ion-text-center">
+              <!-- <ion-label style="color: rgb(85, 84, 84);font-size:7pt !important ;">{{ "DAILY_RECORD.bodyTemperature" | translate }}</ion-label> -->
+              <ion-label style="color: rgb(85, 84, 84);font-size:10pt !important ;">Body Temperature</ion-label>
+              <ion-label style="color: rgb(85, 84, 84);font-size:12 pt !important ;">體溫:</ion-label>
```
Comment (bafu) [inline src/app/daily/daily-overview/daily-overview.component.html:29]: @CS6 Hard-coding English and Mandarin will cause i18n issue.

Comment (CS6) [reply src/app/daily/daily-overview/daily-overview.component.html:29]: 這邊不是要固定顯示嗎？ 我不是很確定，是否等明晚 UI 討論我在確認一次
https://docs.google.com/presentation/d/1tGQ_2rPN_VCFr9LzV1O-gQF-zIV5Q59ANiljWAVQhCo/edit#slide=id.g842fae12e3_0_72

Comment (bafu) [reply src/app/daily/daily-overview/daily-overview.component.html:29]: 上次討論結果是「BT"」看不出是體溫的意思，改成依語系顯示完整的「體溫 / Body Temperature / ...」，這是我的理解

**Comment by CS6** [comment]
@bafu  fixed

**Comment by bafu** [comment]
warm reminder: 顯示相關的 pull request for beta3，只要有支援中英我都會先收

Dixon 建議「多語系支援如果遇到沒有支援的文字，預設顯示英文」

目前還沒有實做，所以日文有些顯示會有問題，例如首頁的「體溫 / Body Temperature」

**Pull request merged**